packages/web-service/src/pages/contact/additional-contacts/additional-ecologist/add-additional-ecologist.njk

```diff
@@ -1,5 +1,5 @@
 {% extends "yes-no.njk" %}
-{% set title = 'Do you want to add an alternative ecologist contact?' %}
+{% set title = 'Do you want to add an alternate ecologist?' %}
 {% set radioHint = 'This alternative contact must be someone who is authorised to deal with this application on behalf of the named ecologist.' %}
```

Comment:
AC4 in the ticket states that this hint text needs to be changed to `This person will not hold the licence, but will receive messages about this application.`

Reply:
Will do. Well spotted. I honestly didn't see that at all